pkg/helmexec/exec.go

```diff
@@ -344,6 +347,18 @@ func (helm *execer) DecryptSecret(context HelmContext, name string, flags ...str
 	return tmpFileName, err
 }
 
+func tempDir(pattern string) (string, error) {
+	workDir := os.Getenv("HELMFILE_TEMPDIR")
+	if workDir == "" {
+		var err error
+		workDir, err = ioutil.TempDir(os.TempDir(), pattern)
+		if err != nil {
+			return "", err
+		}
+	}
+	return workDir, nil
+}
```

Comment:
I guess it would be even more nicer if you could also refactor the below:

https://github.com/roboll/helmfile/blob/ae942c5288895c84c79171e5446773e4cb41c4ce/pkg/state/temp.go#L35-L41

to use this `tempDir` func as well!

Reply:
Yes I saw that, but it's in a different pkg and will cause cyclic import. I will try to put to another pkg such as utils, how do you think?

Reply:
Another pkg sounds good! If I were you I'd name it `pkg/runtime` because I imagine there would be a lot of runtime-environment specific code that are scattered across the code-base now.

Reply:
@mumoshu I renamed the package, but now CircleCI gives me errors that I don't understand, if you have some time, could you help me.